_includes/docs/user-guide/ui/slack-settings.md

```diff
@@ -0,0 +1,30 @@
+* TOC
+{:toc}
+
+By integrating Slack with Thingsboard, users will be able to receive notifications in Slack of events occurring in the Thingsboard system according to the notification rules you set. For example, to notify about device status or detected issues.
```

Comment:
Add a link to the notification rules. The notification documentation draft is already in the "develop/3.5" branch.